docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md

```diff
@@ -58,6 +58,14 @@ Follow these steps to create an EKS Anywhere cluster.
 
    The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/). Cluster creation will succeed if authentication is not set up, but some warnings may be genered.  Detailed package configurations can be found [here]({{< relref "../../tasks/packages" >}}).
 
+     {{% alert title="Note" color="primary" %}}
```

Comment:
Why is this coming back? This is not correct if it has to come back.

Reply:
My mistake. I was working from an old branch. You mean I should remove the whole note, right? I went ahead and did that.